exampleCourse/questions/template/checkbox/random-prompt/question.html

```diff
@@ -0,0 +1,9 @@
+<pl-question-panel>
+  <p>Select all of the following numbers that are {{params.concept}}.</p>
+</pl-question-panel>
+
+<pl-checkbox answers-name="ts" number-answers="7" fixed-order="true" partial-credit="true" min-correct="2">
```

Comment:
Is there any kind of reasonable standard for `answers-name` values? Surely we can do better than `ts`, but our other questions use either `ans` or something more meaningful like `dimension` or `color`. For questions where we don't care about this value (that is, we don't read out of `correct_answers` or anything else in `data`), is a generic `ans` preferable?

Reply:
I like the idea of using a generic `ans` in cases like this. Maybe even `ans1` to make it clear that you need to increment the number if you have multiple elements?

Reply:
I'll stick with `ans` for now.